packages/quantic/cypress/page-objects/search.ts

```diff
@@ -90,6 +94,11 @@ export const InterceptAliases = {
     UndoQuery: uaAlias('undoQuery'),
     SearchboxSubmit: uaAlias('searchboxSubmit'),
   },
+  NextAnalytics: {
```

Comment:
ok this is where we will add `InterceptAliases` for all the new protocol events? should we create its own alias object instead for clarity or do we need to add it to the existing object of `InterceptAliases` ?

Reply:
We do want in the same object, cause the logic to intercept queries is handled at the same place so it would make sense to have only one object that this logic will need to look at.